pkg/api/meta/meta.go

```diff
@@ -28,6 +29,41 @@ import (
 	"github.com/golang/glog"
 )
 
+func ListAccessor(obj interface{}) (List, error) {
```

Comment:
ListAccessor to meta.List is what Accessor to meta.Object.

The UnstructuredList doesn't have a `TypeMeta` field, so it cannot be interpreted by Accessor. However, it implements the List interface, so it could be interpreted by ListAccessor.